charmcraft/application/commands/store.py

```diff
@@ -1897,11 +1912,16 @@ def run(self, parsed_args):
             resource_filepath_is_temp = True
             resource_type = ResourceType.oci_image
 
+            image_arch = image_info.get("Architecture", "all")
+            image_arch = utils.ARCH_TRANSLATIONS.get(image_arch, image_arch)
+            bases = [{"name": "all", "channel": "all", "architectures": [image_arch]}]
```

Comment:
I am not sure the `"name": "all", "channel": "all"` referred to. Should it be the charm name and channel?

Reply:
The store uses bases that are roughly the same as charmcraft's bases, but we currently only care about the architecture. So setting name and channel to "all" allow it to not worry about, for example, whether it's running on Ubuntu 22.04.